 gitignore.txt

```diff
@@ -0,0 +1 @@
+parakh-new/include/constants.php
```

Comment:
This is incorrect file. It should be .gitignore. Also, you need not to commit this file.